icu4j/perf-tests/pom.xml

```diff
@@ -15,6 +15,7 @@
 
   <properties>
     <module-name>perf_tests</module-name>
+    <jmh.version>1.37</jmh.version>
```

Comment:
Please declare this in the `icu4j/pom.xml`, together with the versions for all other dependencies.